IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl/.cproject

```diff
@@ -106,7 +112,9 @@
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\ge_low_mem.obj&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\ge_operations.obj&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\hash.obj&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\hc128.obj&quot;"/>

```

Comment:
We have recently removed HC-128 from wolfSSL.  Can you rebase and remove hc128.c references?

Reply:
Removed the reference.